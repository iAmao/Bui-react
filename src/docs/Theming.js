import React from 'react';


export default () => {
    return (
        <div>
            <h2>Theming</h2>
        </div>
    )
}