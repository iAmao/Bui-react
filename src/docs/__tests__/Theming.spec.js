import React from 'react';
import renderer from 'react-test-renderer';

import { shallow } from 'enzyme';

import Theming from '../Theming';


describe('DOCS: Back', () => {
    describe('Snapshot Test', () => {
        it('renders the theming page', () => {
            const tree = renderer.create(
                <Theming />
            ).toJSON();
            expect(tree).toMatchSnapshot();
        });
    });

    describe('Unit Test', () => {
        it('should render a page showing instructions on theming', () => {
            const wrapper = shallow(<Theming />);

            expect(wrapper.text()).toEqual('Theming');
        })
    });
});
