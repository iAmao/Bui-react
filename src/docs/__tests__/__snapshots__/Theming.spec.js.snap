// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DOCS: Back Snapshot Test renders the theming page 1`] = `
<div>
  <h2>
    Theming
  </h2>
</div>
`;
